tests/target/issue-3255.rs

```diff
@@ -0,0 +1,27 @@
+fn foo() {
+    if true { // Sample comment
+        1
+    }
+}
+
+fn foo() {
+    if true {
+        // Sample comment
+        1
+    }
+}
+
+fn foo() {
+    if true {
+        /* Sample comment */
+        1
+    }
+}
+
+fn foo() {
+    if true {
+        /* Sample
+        comment */
+        1
```

Comment:
personally, I think it makes more sense to not move the comment one line down. For example, the following use cases
```
if true { /* this is a supppppppppppppppppppppppppppper 
   long comment to describe the if condition */
   1
}
```